app/services/auto-approval/checks/do-expenses-match-first-time-claim.js

```diff
@@ -2,24 +2,29 @@ const groupExpensesByType = require('../../notify/helpers/group-expenses-by-type
 const AutoApprovalCheckResult = require('../../domain/auto-approval-check-result')
 
 const CHECK_NAME = 'do-expenses-match-first-time-claim'
-const FAILURE_MESSAGE = 'The number or type of expenses for this claim don\'t match the last manually approved claim'
+const FAILURE_MESSAGE = "The number or type of expenses for this claim don't match the last manually approved claim"
 
 module.exports = function (autoApprovalData) {
   if (autoApprovalData.latestManuallyApprovedClaim && autoApprovalData.latestManuallyApprovedClaim.claimExpenses) {
-    const groupedFirstTimeClaimExpenses = groupExpensesByType(autoApprovalData.latestManuallyApprovedClaim.claimExpenses)
+    const groupedFirstTimeClaimExpenses = groupExpensesByType(
+      autoApprovalData.latestManuallyApprovedClaim.claimExpenses,
+    )
     const groupedCurrentExpenses = groupExpensesByType(autoApprovalData.ClaimExpenses)
 
-    for (let i = 0; i < Object.keys(groupedCurrentExpenses).length; i++) {
+    for (let i = 0; i < Object.keys(groupedCurrentExpenses).length; i += 1) {
       const index = Object.keys(groupedCurrentExpenses)[i]
       const currentExpenseTypeGroup = groupedCurrentExpenses[index]
 
       // For each expense type, check that the first time claim contains the same expense type
       // with the same or less number of claim expenses
       if (!groupedFirstTimeClaimExpenses[index]) {
-        const ADDITIONAL_INFO = `. Claim ref: ${autoApprovalData.Claim.Reference}, No. of current expense type: ${currentExpenseTypeGroup.length}, No. of first time claim expense: ${groupedFirstTimeClaimExpenses[index]?.length ?? 0}`
+        const ADDITIONAL_INFO = `. Claim ref: ${autoApprovalData.Claim.Reference}, No. of current expense type: ${currentExpenseTypeGroup.length}, No. of first time claim expense: 0`
```

Comment:
In the `if` it's already checked so there can't be any first time claim expenses.